### DISCUSSION TITLE
Grouped scheduled action and updates summary

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#672] [Q&A (Community Support)]
Hi, is it possible to configure that the report is received in one email for all pages and not for each page separately?
I have in mind grouped all action in one email and updates in one email.
Thank you

**Comment by nikosdion** [comment]
No, these are strictly per site.

If you want to see all sites at once you'll have to do it from within Panopticon itself. It's the default state of the corresponding views.
